frontend/amundsen_application/api/__init__.py

```diff
@@ -1,20 +1,22 @@
 # Copyright Contributors to the Amundsen project.
 # SPDX-License-Identifier: Apache-2.0
 
-from typing import Any, Tuple
+from typing import Any
 import logging
 
 from flask import Flask, render_template
 import jinja2
 import os
 
+from .healthcheck import run_healthcheck
```

Comment:
any reason this is not absolute import?

Reply:
not particularly, I'll update this to conform